### PR TITLE
Removable Filters Acceptance Tests

### PR DESCRIPTION
**Pull request by oshi97**
This commit adds acceptance tests for Removable Filters for Facets, FilterBox, and RADIUS_FILTER inside a FilterBox. It also adds test for queries like "Consulting" or "Technology", which respond with an NLP Filter as well as check off a facet.

This commit also updates acceptance tests to use their own experience key, sdkacceptancetests, instead of rosetest, and also refactors acceptancesuite.js into separate files. It also removes the FacetsPage object, since conceptually speaking a FacetsPage is just a VerticalPage with Facets on it.
J=SPR-2434
TEST=auto

### CODE REVIEW

File: tests/acceptance/acceptancesuite/universalsuite.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
+import UniversalPage from '../pageobjects/universalpage';
```
Comment (oshi97) [inline tests/acceptance/acceptancesuite/universalsuite.js:1]: this file was spliced out of acceptancesuite.js, and no changes were made

File: tests/acceptance/acceptancesuite/verticalsuite.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
+import VerticalPage from '../pageobjects/verticalpage';
```
Comment (oshi97) [inline tests/acceptance/acceptancesuite/verticalsuite.js:1]: this file was spliced out of acceptancesuite.js, and no changes were made

File: tests/acceptance/fixtures/html/removablefilters copy.html [hunk 3]
```diff
@@ -0,0 +1,143 @@
...
+
+<body>
+  <div class="search-bar"></div>
+  <div class="flex">
+    <div>
+      <div class="facets"></div>
+      <div class="filter-search"></div>
+      <div class="filter-box"></div>
+    </div>
+    <div">
```
Comment (alexisgrow) [inline tests/acceptance/fixtures/html/removablefilters copy.html]: Extra " here!

File: tests/acceptance/fixtures/html/facets.html [hunk 4]
```diff
@@ -19,7 +19,7 @@
   <script>
     ANSWERS.init({
       apiKey: 'df4b24f4075800e5e9705090c54c6c13',
-      experienceKey: 'rosetest',
+      experienceKey: 'sdkacceptancetests',
```
Comment (alexisgrow) [inline tests/acceptance/fixtures/html/facets.html:22]: 😍

File: tests/acceptance/blocks/resultsheadercomponent.js [hunk 5]
```diff
@@ -0,0 +1,26 @@
+import { Selector, t } from 'testcafe';
+
+/**
+ * This class models user interactions with the {@link ResultsHeaderComponent}.
+ */
+export default class ResultsHeaderComponentBlock {
+  constructor (selector) {
+    this._container = '.yxt-ResultsHeader';
+    this._selector = selector || Selector(this._container);
```
Comment (alexisgrow) [inline tests/acceptance/blocks/resultsheadercomponent.js:9]: Is `this._selector` a CSS selector or a DOM element / node?

Comment (oshi97) [reply tests/acceptance/blocks/resultsheadercomponent.js:9]: ah so it's a test cafe selector, which I've been thinking of as a lazy DOM element but not sure if that's 100% accurate. https://devexpress.github.io/testcafe/documentation/guides/basic-guides/select-page-elements.html I think you can pass it either a DOM element or a css selector!

File: tests/acceptance/fixtures/html/removablefilters copy.html (hunk 3 above)
Comment (oshi97) [reply tests/acceptance/fixtures/html/removablefilters copy.html]: oh shoot I shouldn't even have this file. Fixed this in the other removablefilters.html though!

**Pull request closed without merging**